go.mod

```diff
@@ -9,11 +9,11 @@ require (
 	github.com/aristanetworks/goarista v0.0.0-20200609010056-95bcf8053598 // indirect
 	github.com/centrifuge/go-substrate-rpc-client/v2 v2.0.1
 	github.com/ethereum/go-ethereum v1.9.25
-	github.com/google/addlicense v0.0.0-20210428195630-6d92264d7170 // indirect
 	github.com/gorilla/websocket v1.4.2 // indirect
 	github.com/prometheus/client_golang v1.4.1
 	github.com/rs/cors v1.7.0 // indirect
 	github.com/stretchr/testify v1.7.0
 	github.com/urfave/cli/v2 v2.3.0
-	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c // indirect
```

Comment:
Why did you remove this?

Reply:
It was removed automatically by the command rub `go mod tidy`